plugins/outputs/timestream/timestream.go

```diff
@@ -377,6 +386,50 @@ func (t *Timestream) createTable(tableName *string) error {
 	return nil
 }
 
+// Parse user input CreateTableCompositePartitionKey into Timestream partition key list
+// Dimension type partition key requires 3 string elements:
+//  1. "Dimension" to indicate its type
+//  2. User specified composite_partition_key_dimension_name
+//  3. Enforcement level can be "true" or "false"
+//
+// Measure type partition key requires 1 string element:
+//  1. "Measure_name" to indicate its type
+func (t *Timestream) getCompositePartitionKeyList() ([]types.PartitionKey, error) {
+	compositePartitionKeyList := make([]types.PartitionKey, 0, len(t.CreateTableCompositePartitionKey))
+	if len(t.CreateTableCompositePartitionKey) > 0 {
```

Comment:
no need to check the length just use the for loop to do that and un-indent the code